### DISCUSSION TITLE
Make a video to preview all features

### DISCUSSION THREAD

**Discussion by Feel-ix-343** [Feel-ix-343/markdown-oxide#28] [Ideas]
The features list is helpful, but I think that a gif/video showing all of the features off would be better.

**Comment by Idobenhamo** [comment]
Any updates on that?

**Reply by Feel-ix-343** [reply]
Making it today!

**Reply by Idobenhamo** [reply]
Thanks!!!
Any updates on that?

**Reply by Feel-ix-343** [reply]
Damn I'm sorry! I haven't done it. 

I think I am going to make separate gifs for each feature, given that there are changes all the time and I don't want to make a new video after every change. 

I will surely make this by the end of this week. I am trying hard to get the config (+ daily note) support rolled out too

**Reply by Idobenhamo** [reply]
Thank you for your hard work man!

**Reply by Feel-ix-343** [reply]
Hey! This should be all set! Check the readme!

**Reply by Idobenhamo** [reply]
Thank you, this is perfect! By the way, I couldn't figure out how to install the vscode extension. It doesn't seem to be available on the vscode store.
Also, wanted to ask why the installation with cargo doesn't have `--locked`?

**Reply by Feel-ix-343** [reply]
Hi!

Right now its not yet available in the vscode store but you can follow the directions on the readme to test it. I guess I should work on publishing it soon!

And for the installation, I think that would certainly make more sense. Have you noticed a difference though?

**Reply by Idobenhamo** [reply]
As for the the `--locked` this flag insures cargo is installing the specified dependencies and not the latest as far as I know. I think it would be safer this way...

**Reply by Feel-ix-343** [reply]
Updated

**Reply by Idobenhamo** [reply]
Thanks!
I started on learning the syntax & usage of rust BTW. What resources should I use to learn about the lsp? what did you use/recommend?
